custom/icds_reports/templates/icds_reports/icds_app/reassignment-message.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-info" ng-if="selectedLocation.deprecated_at">
```

Comment:
aha! this is better.

nit: i think we should check on `deprecated_by` and `deprecates` even though its true that if those are set, the timestamp fields would be too.

Reply:
I am surprised, we aren't using the operation available on the location?
I see value in telling the user what operation was performed like
`It was merged extracted from ABC on 1-1-2001`

Reply:
> nit: i think we should check on deprecated_by and deprecates even though its true that if those are set, the timestamp fields would be too.

I did it this way, in case the field of what it was replaced with is not properly set. The date determines whether or not it was deprecated. If we can't figure out what it was deprecated to (somehow the location got deleted or something else) it should not prevent us from letting the user know they are viewing an invalid location.